README.md

```diff
@@ -93,11 +93,11 @@ You can set the `enabled` property to false to disable the sticky behavior:
 
 #### CSS
 
-The sticky element has a `sticky-element` class you can use for styling. Furthermore it adds some additional classes
+The sticky element has a `.sticky-element` class you can use for styling, and a `containerClassName` property to provide additional classes. Furthermore it adds some additional classes
 when being sticky:
-* `sticky-element--sticky` (either to the top or the bottom)
-* `sticky-element--sticky-top`
-* `sticky-element--sticky-bottom`
+* `.sticky-element--sticky` class and `containerStickyClassName` property (when sticked either to the top or the bottom)
```

Comment:
I wonder do we really need to have both classes? 

I would suggest to only use `containerStickyClassName` and all the other dynamic class names you introduced, but make them default to the previously used static ones (instead of `null` as they are now). This way the change remains backwards compatible (i.e. same default classes are applied), but if you *want* to customize them, you can do so now. Keeps the implementation simpler I think (no need for `concat`)

Does that make sense?

Reply:
This makes sense. But in this case you cannot apply defaults classes to all sticky elements, _and_ a particular class to one particular sticky element. It's up to you, whether you want to support or not this scenario.

Reply:
I guess if you use a space-separated string of class names, you *could* make it work. But I would say let's not care too much about this, having a customizable class name with the backwards compatible defaults would be fine for me!

Reply:
So is it ready to merge ?

Reply:
Ok, I probably did not communicate clearly. I would still prefer what I suggested in the first comment, to only use the dynamic class name bindings, but with the previous static names as their defaults! 😉

Reply:
Ok, i'll update that